doc/onboarding.md

```diff
@@ -200,7 +200,9 @@ onboarding session.
 * Collaborators are in alphabetical order by GitHub username.
 * Optionally, include your personal pronouns.
 * Label your pull request with the `doc` subsystem label.
-* Run CI on your PR.
+* Run CI on the PR. Because the PR does not affect any code, use either the
+  `node-test-pull-request-lite` CI task or else cancel the CI run after the
```

Comment:
Can you explicitly mention `node-test-pull-request` in the sentence as the alternative, just to avoid being ambiguous

Reply:
Sure, done!